index.html

```diff
@@ -19,13 +19,22 @@
   <header id="header">
     <h1>Tapestry</h1>
     <h2>Intercultural Understanding</h2>
+    <div id="slider-wrapper">
+      <input type="range" min="1" max="3" value="2" class="slider" id="myRange">
+    </div>
+
```

Comment:
There is a lot of empty spaces added in and I'm not sure why. Could you remove them if it's not for any specific reason?

Reply:
resolved by createDepthSlider @ line 150